osu.Game.Rulesets.Mania/UI/Column.cs

```diff
@@ -136,6 +136,24 @@ public void OnReleased(KeyBindingReleaseEvent<ManiaAction> e)
         {
         }
 
+        private ManiaInputManager.RulesetKeyBindingContainer keyBindingContainer { get; set; }
+
+        private ManiaInputManager.RulesetKeyBindingContainer getKeyBindingManager()
```

Comment:
method name doesn't match returned class name (returned type is a container, method called mentions a manager)